docs/guides/multisite_custom.rst

```diff
@@ -0,0 +1,230 @@
+                "meta_tags": meta_tags(
+                    meta_data_providers,
+                    request=request,
+                    site_name=page.site.name
+                )
+            },
+        )
+
+Make sure that your custom site model get registered with the Django ORM
+before your page model gets registered, otherwise there will be an exception
+telling you that the site model configured in ``FEINCMS3_SITES_SITE_MODEL``
+does not exist.
+
+Remember to update your migrations as well:
+
+.. code-block:: shell
+
+    ./manage.py makemigrations
+    ./manage.py migrate
+
+
+Implementing a Custom Site Model Admin
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+    from django import forms
+    from django.conf import settings
+    from django.contrib import admin
+    from django.utils.translation import gettext_lazy as _
+
+    from feincms3_meta.models import MetaMixin
+
+    from .models import Site
+
+
+    class SiteForm(forms.ModelForm):
+        default_language = Site._meta.get_field("default_language").formfield(
+            choices=[("", "----------")] + list(settings.LANGUAGES),
+        )
+
+        class Meta:
+            model = Site
+            fields = "__all__"
+
+
+    class SiteDefaultLanguageListFilter(admin.SimpleListFilter):
+
+        title = _("default language")
+        parameter_name = "default_language"
+
+        def lookups(self, request, model_admin):
+            return settings.LANGUAGES
+
+        def queryset(self, request, queryset):
+            if self.value():
+                return queryset.filter(default_language=self.value())
+            else:
+                return queryset.all()
+
+
+    @admin.register(Site)
+    class SiteAdmin(admin.ModelAdmin):
+        form = SiteForm
+
+        list_display = [
+            "name", "host", "default_language", "is_active", "is_default"
+        ]
+
+        list_filter = [
+            "is_active", "host", "name", SiteDefaultLanguageListFilter
+        ]
+
+        fieldsets = [
+            ("Standard", {
+                "fields": [
+                    "name",
+                    "default_language",
+                    "is_active",
+                    "is_default",
+                ],
+                "classes": [
+                    "tabbed",
+                    "uncollapse"
+                ],
+            }),
+            ("Host", {
+                "fields": [
+                    "host",
+                    "is_managed_re",
+                    "host_re",
+                ],
+                "classes": [
+                    "tabbed"
+                ],
+            }),
+            MetaMixin.admin_fieldset()
+        ]
+
+By default, the admin edit/create page will display itself as a flat admin page,
+i.e. there will not be any tabs.
+
+In comes the ``django-content-editor`` and the scripts and other media it provides.
+
+So to have tabs on your custom site model's admin page, simply add the
```

Comment:
```suggestion
So to have tabs on your custom site model's admin page, add the
```

Please avoid words such as `simply`; it may not be simple to newcomers etc. 

Here's a good rationale for this: https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#writing-style